Jenkinsfile

```diff
@@ -314,7 +314,7 @@ pipeline {
           }
           stage('Playstore') {
             when {
-              expression { env.trackName != 'None' && env.flavor != 'Dev' && env.CHANGE_ID == null }
+              expression { env.trackName != 'None' && env.CHANGE_ID == null }
```

Comment:
This will make it upload to play store on every branch 
```groovy
def defineFlavor() {
    //check if the pipeline has the custom flavor env variable set
    def overwrite = env.CUSTOM_FLAVOR
    if(overwrite != null) {
        return overwrite
    }
    def branchName = env.BRANCH_NAME
    if (branchName == "main") {
        return 'Internal'
    } else if(branchName == "develop") {
    } else if (branchName == "develop") {
        return 'Dev'
    } else if(branchName == "release") {
    } else if (branchName == "release") {
        return 'Public'
    }
    return 'Dev' // 'Dev' here is the defualt flaver name 
}
```